x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/index.tsx

```diff
@@ -246,7 +197,13 @@ const ConfigureCasesComponent: React.FC<ConfigureCasesComponentProps> = ({ userC
 
   useEffect(() => {
     handleActionBar();
-  }, [connectors, connectorId, closureType, currentConfiguration]);
+  }, [
+    connectors,
+    connectorId,
+    closureType,
+    currentConfiguration.connectorId,
```

Comment:
If we just use `currentConfiguration`, the tests don't pick up the updates and fail, listening to the values works